applications/nrf5340_audio/doc/configuration.rst

```diff
@@ -85,3 +85,87 @@ You can use one of the following options, depending on how you decide to build t
 To set the TX power output, use the ``CONFIG_BT_CTLR_TX_PWR_ANTENNA`` and ``CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB`` Kconfig options in :file:`applications/nrf5340_audio/sysbuild/ipc_radio/prj.conf`.
 
 See :ref:`ug_radio_fem` for more information about FEM in the |NCS|.
+
+.. _nrf53_audio_app_configuration_sd_card_playback:
+
+Enabling SD card playback
+*************************
+
+The SD Card Playback module allows you to play audio files directly from an SD card inserted into the nRF5340 Audio development kit.
+This feature supports both WAV and LC3 audio file formats and is compatible with all nRF5340 Audio applications.
+
+File format support requirements
+================================
+
+The SD card playback module supports both WAV and LC3 audio file formats.
+The audio files must meet the following requirements:
+
+* WAV files must be 48 kHz, 16-bit, mono PCM format.
+* LC3 files must be in the LC3 file format with proper headers.
+
+SD card requirements
+====================
+
+Make sure the SD card meets the following requirements:
+
+* Formatted with FAT32 or exFAT file system.
+* Audio files are placed in the root directory or subdirectories.
+
+Configuring SD card playback
+============================
+
+To enable SD card playback functionality, you need to set the following Kconfig options to ``y``:
+
+* ``CONFIG_NRF5340_AUDIO_SD_CARD_MODULE`` - to enable the SD card module; this option is enabled by default on nRF5340 Audio DK
+* ``CONFIG_SD_CARD_PLAYBACK`` - to enable the playback functionality
```

Comment:
Aren't these and the following application specific Kconfigs? If yes,  should be described like https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/templates/application_README.html#configuration_options